.tern-project

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
@aminroosta This is bit personalized. I use webstorm and it works perfectly fine. It has superior auto complete feature and unfortunately, it is not free. please remove this. We dont need it. For code analysis, we should use eslint. We are already ES6, we should go the ESLINT rout